data/json/monsters/fungus_zombie.json

```diff
@@ -1,4 +1,71 @@
 [
+  {
+    "id": "mon_zombie_dusted",
+    "type": "MONSTER",
+    "name": { "str": "dusted zombie" },
+    "description": "This shambling corpse has been heavily dusted with a thick gray powder.",
+    "default_faction": "zombie_dusted",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 45,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "light_gray",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 3,
+    "melee_dice": 1,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "vision_day": 10,
+    "vision_night": 3,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_gross_anatomy", "prof_intro_biology", "prof_physiology", "prof_wp_zombie" ],
+    "harvest": "zombie_humanoid",
+    "grab_strength": 20,
+    "special_attacks": [
+      [ "PARROT", 400 ],
+      [ "PARROT_AT_DANGER", 0 ],
+      { "id": "grab" },
+      { "id": "bite_humanoid", "cooldown": 5 },
+      { "id": "scratch_humanoid" }
+    ],
+    "death_drops": "default_zombie_death_drops",
+    "burn_into": "mon_zombie_scorched",
+    "upgrades": { "age_grow": 1, "into": "mon_zombie_fungalize" },
+    "flags": [
+      "SEES",
+      "HEARS",
+      "STUMBLES",
+      "WARM",
+      "GRABS",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ],
+    "armor": { "electric": 1 }
+  },
+  {
+    "id": "mon_zombie_fungalize",
+    "type": "MONSTER",
+    "name": { "str": "fungal stalk zombie" },
+    "description": "This writhing corpse has grown a short stalk of gray fungus bursting from the skull.",
+    "copy-from": "mon_zombie_fungus",
+    "diff": 0,
+    "hp": 80,
+    "speed": 20,
+    "bleed_rate": 50,
+    "special_attacks": [ { "type": "bite", "cooldown": 5 } ],
+    "upgrades": { "age_grow": 1, "into": "mon_zombie_fungus" },
+    "flags": [ "SEES", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "PUSH_MON", "FILTHY" ]
```

Comment:
Is there a reason that it's no longer able to hear?

Reply:
I copied that from the final "mon_zombie_fungus" version. There my guess is that the ear canal is stuffed full and doesn't work anymore. I don't think that makes as much sense here. I'm restoring hearing for this step.